fm.cancel.Ripcord.yaml

```diff
@@ -0,0 +1,38 @@
+app-id: fm.cancel.Ripcord
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+finish-args:
+  - --socket=x11
+  - --socket=wayland
```

Comment:
does it support wayland?

Reply:
you didn't answer the question.

Reply:
oh yeah, not it doesn't, I forgot to add it to the previous commit